python/device/iothub-client-python/windows/iothub_client_python.vcxproj

```diff
@@ -0,0 +1,187 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This should be cmake driven from the get go so that we plug it in nicely with the build system.